src/components/Sections/Topics/RenderPage.js

```diff
@@ -27,7 +27,7 @@ const RenderPage = ({ t }) => {
   const breadCrumbSections = [{ id: '', label: t('nav.sidebar.topics') }, ...tagPath].map(getBreadCrumbSection);
 
   const breadCrumbIcon = `${isLanguageRtl(language) ? 'left' : 'right'} angle`;
-
+  const baseParams     = useMemo(() => ({ tag: id }), [id]);
```

Comment:
Why useMemo here?

Reply:
we push it to props of child node, I think its can make additional render of child